chat-ui-manager-app/src/routes/+page.svelte

```diff
@@ -0,0 +1,134 @@
+<script lang="ts">
+	import { onMount } from 'svelte';
+	import { MachineStatus } from '$lib/types';
+	import { env } from '$env/dynamic/public';
+
+	const chatUIUrl = env.PUBLIC_CHAT_UI_URL;
+	let machineStatus: MachineStatus = $state(MachineStatus.UNKNOWN);
+	let chatUIAvailable = $state(false);
+
+	onMount(async () => {
+		await checkMachineStatus();
```

Comment:
You'll probably want to use an `AbortController` on your request and return that from the mount function here, that way if there is a page navigation you won't keep the request going (since a large number of refreshes could then keep a lot of requests open)
https://developer.mozilla.org/en-US/docs/Web/API/AbortController